assembl/static2/js/app/components/common/modifyPasswordForm.jsx

```diff
@@ -35,23 +35,23 @@ class ModifyPasswordForm extends React.Component<void, Props, State> {
     newPassword2: ''
   };
 
-  handleOldPasswordChange = (e: SyntheticEvent) => {
+  handleOldPasswordChange = (e: SyntheticEvent<>) => {
```

Comment:
You normally can use `SyntheticEvent<HTMLInputElement>` and remove the `if (e.target instanceof HTMLInputElement) `